presto-tpch/src/main/java/io/prestosql/plugin/tpch/TpchRecordSet.java

```diff
@@ -109,7 +109,8 @@ public TpchRecordCursor(Iterator<E> rows, TpchTable<E> table, List<TpchColumn<E>
         @Override
         public long getCompletedBytes()
         {
-            return 0;
+            // for test
+            return 10;
```

Comment:
is this needed ?

Reply:
If return 0, the testcase testQueryScanExceeded won't exceed query.max-scan-physical-bytes, even set query.max-scan-physical-bytes to 0B.

Reply:
But this 10 looks a bit arbitrary number, either we have to fetch the completed bytes for a TPCH connector or we can try with a connector which exposes the completedBytes. Can we try with a query like this 
`SELECT * FROM system.runtime.nodes`